stable/bookstack/values.yaml

```diff
@@ -160,3 +160,13 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+## Enable ldap authentication. See https://www.bookstackapp.com/docs/admin/ldap-auth/ for details on how to set it up.
+ldap:
+  enabled: true
```

Comment:
ldap should be disabled by default and the rest of the values should be either a usable default or blank/commented out.

Reply:
Yup, my mistake - it should've been disabled by deafult. Fixed.